app/helpers/external_link_helper.rb

```diff
@@ -128,13 +132,21 @@ def i_belong_action_plan_url
   end
 
   def stem_ambassadors_url
+    "https://www.stem.org.uk/stem-ambassadors"
+  end
+
+  def computing_ambassadors_url
```

Comment:
We had this helper method as stem_ambassadors_url originally. It made sense to rename it to be more relevant. I checked and updated any calls to this throughout the code.

Reply:
Will update this to sanitize